rpc_jobs/rpc_gating_project_builds.yml

```diff
@@ -0,0 +1,29 @@
+- job:
+    name: 'RPC-Gating-Project-Builds'
+    project-type: workflow
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_project_builds.*'
+          only-trigger-phrase: false
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/project_builds'
```

Comment:
shouldn't we do:
```white-list-target-branches: master```?

you said to me:
```If a branch isn’t listed, or isn’t matched by a pattern, then PRs to it will be ignored```.